src/org/omegat/gui/main/MainWindow.java

```diff
@@ -461,9 +462,13 @@ public void displayErrorRB(final Throwable ex, final String errorKey, final Obje
             }
 
             statusLabel.setText(msg);
-            String fulltext = msg;
+            StringBuilder fulltext = new StringBuilder(msg);
             if (ex != null) {
-                fulltext += "\n" + ex.toString();
+                fulltext.append("\n");
+                for (String s : ex.toString().split("\\R")) {
```

Comment:
It might be better to use `getLocalizedMessage()` or `getMessage()` instead of `toString()`, is it?

```suggestion
                for (String s : ex.getLocalizedMessage().split("\\R")) {
```